docs/conf.py

```diff
@@ -309,14 +306,6 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
 
-
-def setup(app):
-    app.add_js_file("js/script.js")
```

Comment:
This reverts #3476. Was that deliberate, or should I look at making script.js work with Furo?

Reply:
Thanks for pointing out that feature.
Furo's way of doing things does mean that there isn't a neat way to implement #3476 anymore.